.drone.yml

```diff
@@ -7,6 +7,7 @@ steps:
 - name: build
   image: golang:1.12
   commands:
+  - apt update && apt install ca-certificates libgnutls30 -y
```

Comment:
this should not be needed (not required by our other repositories) so maybe just needs a Go version upgrade.  Let's investigate this further.

Reply:
Okay. Bumping to 1.17 worked. Thanks!